webservices/args.py

```diff
@@ -223,6 +223,8 @@ def make_seek_args(field=fields.Int, description=None):
 efilings = {
     'file_number': fields.List(fields.Int, description=docs.FILE_NUMBER),
     'committee_id': fields.List(IStr, description=docs.COMMITTEE_ID),
+    'report_id': fields.Int(),
+    'receipt_date': fields.Date(),
```

Comment:
Yes you are right!  I forgot that was is passed as the sort parameter determines the sorting column, with that being said the rest of the parameters can be pulled.